cmd/tink-worker/worker/log_capturer.go

```diff
@@ -0,0 +1,60 @@
+package worker
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/packethost/pkg/log"
+)
+
+// LogCapturer emits container logs.
+type LogCapturer interface {
```

Comment:
What consumes this interface? Nothing in this package seems to so I wonder if we could just export the struct and leave the interface declaration to consumers?

Reply:
Sure, I can make the implementation public. I think I'll still leave the interface in this package unless there is a reason not to.

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#interfaces

Reply:
I'll move the `LogCapturer` and `ContainerManager` interface declarations into `worker.go`. They're still in the same package, but will be declared at the site of use.